bundles/org.openhab.binding.modbus.e3dc/src/main/resources/OH-INF/update/update.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="e3dc:e3dc">
+		<instruction-set targetVersion="1">
+			<add-channel id="battery-charged">
```

Comment:
I believe these channels are part of a channel group? And as such I think this is needed:
```suggestion
			<add-channel id="battery-charged" groupIds="power-values">
```

Just to be sure, was it tested successfully without? Please confirm a successful test since we are getting close to the release, so not much time left for regressions.